metadata_service/api/swagger_doc/badge/badge_get.yml

```diff
@@ -0,0 +1,16 @@
+Get badges
+---
+tags:
+  - 'badge'
+responses:
+  200:
```

Comment:
This gets all badges in the system at any given time so it would either return all badges or if none exist an empty list, do there should not be any 404s